governance/community-calls/20240115-coworking.md

```diff
@@ -0,0 +1,49 @@
+# Notes 15 January 2024
+
+## Attendees
+
+* Anne
+* Batool
+* Emma
+* Bastian
+* Alex
+* Gabin
+* Kirstie
+
+## Topics/Rooms
+
+* Main room
+  * Kirstie
+  * Bastian
+* Bookdash TTW report
+  * Ale
+  * Anne
+  * Batool
+  * Emma
+* Research Community Management paper review
+  * Gabin
+
+ ## Notes
+
+* Kirstie and Bastian chatted in the main room about using VSCode for pull requests
+  * (And decided that we don't really need the text editor to do that!)
+  * Did open up the History PR: https://github.com/the-turing-way/the-turing-way/pull/3159
+      * KW making a branch -> to merge/make PR to existing branch
+* Book Dash TTW report
+    * Reviewing questions and various sections
+    * Adding a section around future planning
+    * https://annuel2.framapad.org/p/bookdash-nov2023-report
+    * Next Monday 22 January, we will upload the PR for Kirstie's and Malvika's review
+    * Who is the audience? Mostly us.
+        * KW: Demonstrating our commitment to transparency. Can we write this report to be useful for newcomers?
+        * EK: Reading reports was useful for new committee members this Book Dash.
+    * Discussing membership in the committee
+        * AAA: What is the right number of people to make strategic decisions? 
+        * GK: 5
+        * BGT: 7
+        * KW: Less about the size, but about the chair (final decision maker)
+	* Delivery team and working group split
+		* Two chairs, delivery team needs support
+* Gabin - review paper (need to print it out)
```

Comment:
This was me attempting to take notes - clearly I repeated myself, if you want to change this at all!